agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2302,6 +2317,13 @@ bool backhaul_manager::handle_multi_ap_policy_config_request(ieee1905_1::CmduMes
         ap_metrics_reporting_info.reporting_interval_s =
             metric_reporting_policy_tlv->metrics_reporting_interval_sec();
         ap_metrics_reporting_info.last_reporting_time_point = std::chrono::steady_clock::now();
+
+        if (ap_metrics_reporting_info.reporting_interval_s != 0) {
+            set_select_timeout(ap_metrics_reporting_info.reporting_interval_s *
```

Comment:
I think that changing the select timeout will have side effects on the rest of actions that have to be executed periodically.

Reply:
Good point! The select timeout is currently set at a fixed 500ms. So there actually is no reason to change it since the reporting interval is going to be more than 1s. So @VladyslavTupikin please remove the set_select_timeout and add a comment that it's not needed since the timeout is 500ms.

Reply:
Fixed, added comment.